samples/besu/besu-cli/src/commands/asset/lock.ts

```diff
@@ -0,0 +1,140 @@
+import { GluegunCommand } from 'gluegun'
+import { getNetworkConfig, commandHelp } from '../../helper/helper'
+import { getContractInstance } from '../../helper/besu-functions'
+const Web3 = require ("web3")
+const crypto = require('crypto')
+
+const command: GluegunCommand = {
+	name: 'lock',
+  
+	run: async toolbox => {
+		const {
+			print,
+			parameters: { options }
+		} = toolbox
+		if (options.help || options.h) {
+			commandHelp(
+				print,
+				toolbox,
+				`besu-cli asset lock --network=network1 --token_contract='path/to/tokenContract.json' --interop_contract='path/to/interopcontract.json' --sender_account=1 --recipient_account=2 --amount=5 --timeout=10`,
+				'besu-cli asset lock --network=<network1|network2> --token_contract=<path-to-tokenContract.json> --interop_contract=<path-to-interopContract.json> --sender_account=<1|2> --recipient_account=<2|1> --amount=<lock-amount> --timeout=<lock-duration-seconds> --hash=<hashLock-optional-parameter>',
+				[
+					{
+						name: '--network',
+						description:
+							'network for command. <network1|network2>'
+					},
+					{
+						name: '--token_contract',
+						description:
+							'Path to the json file corresponding to the token contract compiled with Truffle.'
+					},
+					{
+						name: '--interop_contract',
+						description:
+							'Path to the json file corresponding to the interop contract compiled with Truffle.'
+					},
+					{
+						name: '--sender_account',
```

Comment:
Do we need this, given that we are setting it in `config.json`.

Reply:
Alternatively We can remove sender and receiver from `config.json`.

Reply:
I thought the accounts can be taken from the config for the exchange command and that other commands can take them via command line. What do you think?

If you don't favor that, I will go with taking from config.json for all the commands.